modules/covid_research/app/services/covid_research/volunteer/form_crypto_service.rb

```diff
@@ -3,28 +3,26 @@
 module CovidResearch
   module Volunteer
     class FormCryptoService
-      extend AttrEncrypted
```

Comment:
For the attr_encrypted approach to encrypted strings, you had to pass around the iv, but this is now not the case with lockbox. CovidResearch:: Volunteer:: FormCryptoService is not an ActiveRecord class so I followed the instructions [here](https://github.com/ankane/lockbox#strings) for string encryption